### PR TITLE
add support for python 3.10

### PR DESCRIPTION
**Pull request by arroyc**
- [x] The purpose of this PR is explained in this message or in an issue. 
- [x] Tests are included and/or updated for code changes. >> will add version tests shortly
- [x] Proper license headers are included in each file.

### CODE REVIEW

File: platforms/python/buildPython.sh [hunk 1]
```diff
@@ -18,11 +18,20 @@ mkdir -p "$targetDir"
 
 builtPythonPrereqs=false
 buildPythonPrereqsImage() {
+    debianType=$debianFlavor
+	# stretch is out of support for python, but we still need to build stretch based
+	# binaries because of static sites, they need to move to buster based jamstack image
+	# before we can remove this hack
```
Comment (william-msft) [inline platforms/python/buildPython.sh:24]: @arroyc may you elaborate more on this? Or link to the action item please

Comment (arroyc) [reply platforms/python/buildPython.sh:24]: https://devdiv.visualstudio.com/DevDiv/_boards/board/t/Oryx/Stories/?workitem=1428737

**Review by william-msft — APPROVED** [review]

File: platforms/python/prereqs/build.sh [hunk 2]
```diff
@@ -10,14 +10,11 @@ wget https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_
 wget https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc -O /python.tar.xz.asc
 
 debianFlavor=$DEBIAN_FLAVOR
+debianHackFlavor=$DEBIAN_HACK_FLAVOR
 pythonSdkFileName=""
+PYTHON_GET_PIP_URL="https://github.com/pypa/get-pip/raw/3cb8888cc2869620f57d5d2da64da38f516078c7/public/get-pip.py"
```
Comment (cormacpayne) [inline platforms/python/prereqs/build.sh:15]: Is this URL (and the SHA variable down below) tied to a specific version of pip? Do these two variables need to be updated whenever we updated the version of pip used in the `__pythonVersions.sh` script?

**Comment by william-msft** [comment]
Closing. Please refer to:
https://github.com/microsoft/Oryx/pull/1200
https://github.com/microsoft/Oryx/pull/1202

**Pull request closed without merging**